docs/cards.html

```diff
@@ -0,0 +1,394 @@
+                        </div>
+                        <div class="overlay-text grey-badge">Out of Stock</div>
+                        <div class="card-child card-body">
+                            <hr>
+                            <p class="card-desc  p-sm">Ruled Journal - Hard Bound - A5, 100GSM, 160 Pages</p>
+                        </div>
+                        <div class="card-child card-footer">
+                            <div class="discounted-price">Rs. 299</div>
+                            <div class="actual-price p-sm">Rs. 499</div>
+                            <div class="discount p-sm">60% OFF</div>
+                        </div>
+                    </diV>
+                </div>
+            </div>
+
+            <div class="component-wrapper">
+                <h1 class="wrapper-heading">Cards - Text Only</h1>
+                <hr>
+                <div class="description wrapper-description">
+                    The below cards are only text with no images.
+
+                    <br>
+                    There are 2 variants available: <code class="class-highlight">text-horizontal</code> and <code
+                        class="class-highlight">text-vertical</code>, based on the preferred type of card. Use these
+                    classes along with the class <code class="class-highlight">text-only</code> in the card's container.
+                </div>
+                <div class="component-showcase cards-text-only">
+                    <div class="card-container text-only text-horizontal">
+                        <div class="card-child card-header">
+                            <img class="card-img"
+                                src="https://m.media-amazon.com/images/I/71qLvcSA2lL._AC_UL480_FMwebp_QL65_.jpg" alt="">
+                            <h4 class="card-title">Title </h4>
+                            <p class="card-subtitle p-sm">Subtitle</p-sm>
+                        </div>
+                        <div class="card-badge card-badge-def badge-right">New!!</div>
+                        <div class="card-child card-body">
+                            <hr>
+                            <p class="card-desc  p-sm">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempora
+                                quae sapiente cum. Lorem ipsum dolor sit amet consectetur adipisicing elit. Natus nemo
+                                atque alias velit earum voluptate magni. Eos deserunt cum voluptatibus laborum.
+                                Corrupti, vel nobis incidunt culpa sequi voluptatum totam dicta!</p>
+                        </div>
+                        <div class="card-child card-footer">
+                            <div class="actions-wrapper">
+                                <button class="action-btn p-sm">Read More</button>
+                                <button class="action-btn p-sm">Bookmark</button>
+                            </div>
```

Comment:
the buttons and the badges look similar, try increasing padding so they will look more like buttons

Reply:
Thank you, will work on it.